demos/mkdocs/appyx-interactions/interactions/observemp/web/src/jsMain/resources/styles.css

```diff
@@ -0,0 +1,12 @@
+#root {
+    width: 100%;
+    height: 100vh;
+}
+
+body {
+    margin: 0;
+}
+
+#root > .compose-web-column > div {
+    position: relative;
+}
```

Comment:
Please, make sure to add this media query css statement to make sure sample adapts on mobile devices:
```css

@media (max-width: 511px) {
    #ComposeTarget {
        scale: 0.5;
        transform-origin: 0 0;
    }
}
```